src/components/MaterialUI/NLinesText.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import classNames from "classnames";
+import { TextProps } from "./textProps";
+
+const useStyles = props =>
+	makeStyles(theme => ({
+		lineClamp: {
+			display: "block",
+			position: "relative",
+			overflow: "hidden",
+			textOverflow: "ellipsis",
+			padding: "0 !important",
+			height: "calc(1em * 1.2 * " + props.lineCount + ")",
+			"&::after": {
+				content: "'...'",
+				textAlign: "right",
+				bottom: "0",
+				right: "0",
+				width: "25%",
+				display: "block",
+				position: "absolute",
+				height: "calc(1em * 1.2)",
+				background:
+					"linear-gradient(to right, rgba(255, 255, 255, 0), rgba(255, 255, 255, 1) 75 %)",
```

Comment:
Is that necessary to have linear-gradient in background color here? The color for "dots" should be the same the color for text and used from theme.